engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -222,28 +227,21 @@ private void standPlayer(EntityRef entity) {
      */
     private void stopAutoMove() {
         isAutoMove = false;
+        List<Input> inputs = bindsConfig.getBinds(new SimpleUri("engine:forwards"));
+        Input forwardKey = inputs.get(0);
```

Comment:
Could be, but why would anyone want that? 😛 
I tested and it throws a NPE but it doesn't really affect the game. It works only if forward is defined. What exactly are the other means of moving forward, if the user has no keyboard binding for forward movement, hypothetically?